tokio/CHANGELOG.md

```diff
@@ -1,3 +1,61 @@
+# 1.41.0 (Oct 19th, 2024)
+
+### Added
+
+- metrics: stabilize `global_queue_depth` ([#6854], [#6918])
+- net: add conversions for unix `SocketAddr` ([#6868])
+- sync: add `watch::Sender::sender_count` ([#6836])
+- sync: add `mpsc::Receiver::blocking_recv_many` ([#6867])
+- task: stabilize `Id` apis ([#6793], [#6891])
+
+### Added (unstable)
+
+- runtime: add `LocalRuntime` ([#6808])
```

Comment:
Should we mention that this is unstable at the moment?

Reply:
It seems like it's already under an "unstable" section?

Reply:
Oh, you're right, I completely missed that! Thanks! 🤦‍♀️